.env.template

```diff
@@ -2,15 +2,3 @@
 MLFLOW_TRACKING_USERNAME=someusername
 MLFLOW_TRACKING_PASSWORD=somepassword
 MLFLOW_TRACKING_SERVER_URI = https://mlflow.endpoint
-
-# If True, experiments already performed and recorded in mlflow won't be rerun.
-USE_CACHE=False
-
-# Parameter tuning method to use:
-# True: use bayesian optimisation
-# False: use grid search
-USE_OPTIMIZATION=False
-
-# Number of simulations to run when using bayesian optimisation for parameter
-# tuning
-OPTIMIZATION_NCALLS=10
```

Comment:
We could add here the vars ELASTICSEARCH_PORT and ELASTICSEARCH_HOSTNAME. I believe they are read from here, no? Just as documentation, to let users know it is an option.

Reply:
Yes, absolutely.